packages/core/src/carousel/private/utils/withHorizontalScroll.ts

```diff
@@ -232,6 +233,32 @@ export const withHorizontalScroll = (options: Options): any => {
     }
   }, []);
 
+  const handleMouseDown = () => {
+    setIsDragging((prevState) => !prevState);
+  };
+
+  const handleMouseUp = () => {
+    setIsDragging((prevState) => !prevState);
+  };
+
+  const debounce = (fallback: (...args: any) => void, delay: number) => {
```

Comment:
Is it possible to avoid using any as type?

Reply:
Not sure in this case. The fallback can be any function with any set of params.